packages/openapi-generator/src/file-serializer/index-file.ts

```diff
@@ -0,0 +1,34 @@
+import { codeBlock, kebabCase } from '@sap-cloud-sdk/util';
+import { OpenApiDocument } from '../openapi-types';
+
+/**
+ * @experimental This API is experimental and might change in newer versions. Use with caution.
+ * Generate the index file to expose the SAP Cloud SDK API and models.
+ * @param openApiDocument Parsed service.
+ * @returns The index file contents.
+ */
+export function exportAllFiles(fileNames: string[]): string {
+  return codeBlock`${fileNames
+    .map(fileName => exportAll(`${kebabCase(fileName)}`))
+    .join('\n')}`;
+}
+
+export function apiIndexFile(openApiDocument: OpenApiDocument): string {
+  const files = [
+    ...openApiDocument.apis.map(api => api.name),
+    ...(openApiDocument.schemas.length ? ['model'] : [])
+  ];
+  return codeBlock`
+    ${exportAllFiles(files)}
+  `;
+}
+
+export function modelIndexFile(openApiDocument: OpenApiDocument): string {
+  return codeBlock`
+    ${exportAllFiles(openApiDocument.schemas.map(schema => schema.name))}
```

Comment:
Does this model work if there are no schemas - see internal issue on services with no schemas.

Reply:
Currently this is prevented by only ever calling this if there are schemas. Do you think this is enough?